### PR TITLE
Fix minimap offset.

### LINKED ISSUE #320: Minimap shows unnecessary whitespace at bottom

### PR DESCRIPTION
**Pull request by jgosmann**
Fixes #320.

### CODE REVIEW

**Comment by cchan987** [comment]
This breaks the minimap for me when I pull netgraph items far apart. The viewport doesn't track properly.

**Comment by jgosmann** [comment]
:wrench: try again

**Comment by Seanny123** [comment]
LGTM.

File: nengo_gui/static/components/netgraph.js [hunk 1]
```diff
@@ -649,11 +649,20 @@ Nengo.NetGraph.prototype.scaleMiniMap = function () {
...
-        var view_offsetX = -(this.mm_min_x + this.offsetX) * w;
-        var view_offsetY = -(this.mm_min_y + this.offsetY) * h;
+        var w = mm_w * this.mm_scale;
+        var h = mm_h * this.mm_scale;
+
+        var disp_w = (this.mm_max_x - this.mm_min_x) * w;
+        var disp_h = (this.mm_max_y - this.mm_min_y) * h;
+
+        var offsetX = -this.mm_min_x * w + (mm_w - disp_w) / 2.;
+        var offsetY = -this.mm_min_y * h + (mm_h - disp_h) / 2.;
```
Comment (tcstewar) [inline nengo_gui/static/components/netgraph.js]: Are these two variables ever used?

Comment (jgosmann) [inline nengo_gui/static/components/netgraph.js]: Doesn't look like it ...?

**Pull request merged**